lib/contexts/index.js

```diff
@@ -3,8 +3,7 @@
  */
 module.exports = {
   'https://www.w3.org/2018/credentials/v1': require('./credentials-v1'),
-  'https://www.w3.org/2018/credentials/examples/v1': require('./vc-examples-v1'),
-  'https://www.w3.org/ns/odrl.jsonld': require('./odrl'),
-  'https://w3id.org/security/v2': require('./security-v2'),
```

Comment:
We removed the security v1 and security v2 context here. Where are their replacements?

Reply:
The replacements for v1 and v2 should be in jsigs, I believe.

Reply:
The terms from security v1 and security v2 have been defined in the credentials-v1 context, flattening it.

Reply:
If they need to be exported here, they should be first pulled in from the `security-context` package.

Reply:
No, we don't want to export them here.